CHANGELOG.md

```diff
@@ -11,7 +11,9 @@
     - Added to remove some code duplication.
  - [ENHANCEMENT] When closing the db any running compaction will be cancelled so it doesn't block.
    - `NewLeveledCompactor` takes a context.
- - [CHANGE] `prometheus_tsdb_storage_blocks_bytes_total` is now `prometheus_tsdb_storage_blocks_bytes`
+ - [CHANGE] `prometheus_tsdb_storage_blocks_bytes_total` is now `prometheus_tsdb_storage_blocks_bytes`.
+ - [BUGFIX] Improved Merge performance.
```

Comment:
Is that really a bugifx?

Reply:
Yes, it fixed a regression from previous performance improvements.

Reply:
Should we name it "Improved Postings Merge performance"? Given that we have query merge now, this can be confusing.

Reply:
updated.